packages/mdc-textfield/_mixins.scss

```diff
@@ -47,9 +47,7 @@
   }
 }
 
-// Private mixins
-
-@mixin mdc-text-field-invalid-label-shake-keyframes_($modifier, $positionY, $scale: .75) {
+@mixin mdc-text-field-invalid-label-shake-keyframes_($modifier, $positionY, $positionX: 0%, $scale: .75) {
```

Comment:
is this change something that should be in this PR? Or unrelated to this PR?

Reply:
Unrelated. This was my merge with master.